backend/src/main/kotlin/cz/loono/backend/schedule/BadgeDowngradeScheduler.kt

```diff
@@ -0,0 +1,15 @@
+package cz.loono.backend.schedule
+
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+
+@Component
+class BadgeDowngradeScheduler(
```

Comment:
Proc jsi vytvoril novy Scheduler a nerozsiril LoonoTaskScheduler? Nemuzu na to prijit. :-)

Reply:
jo, mas recht, fixed in [4698531](https://github.com/cesko-digital/loono-be/pull/43/commits/469853109ab8924b7f56bb60e8e772cba119fdef)